### DISCUSSION TITLE
WS.Reputation.1 Bedrohung

### DISCUSSION THREAD

**Discussion by sygo1** [GrandOrgue/grandorgue#1471] [Help]
In der grandorgue-3.10.1-1.windows.x86_64.zip wird beim Start der grandorgue.exe von NORTON 360 eine Bedrohung mit der Bezeichnung WS.Reputation.1 gemeldet. Die grandorgue.exe wird isoliert und gelöscht.

**Comment by MStraeten** [comment]
eher ein Fehler bei NORTON als bei grandorgue - sofern exe oder zip von hier verwendet wird

**Comment by sygo1** [comment]
Danke für die schnelle Rückmeldung. Der Incident kann geschlossen werden. 

Allerdings kam beim Start auch noch eine Warnmeldung von Microsoft. 

 

Von: Martin Straeten ***@***.*** 
Gesendet: Sonntag, 12. März 2023 16:34
An: GrandOrgue/grandorgue ***@***.***>
Cc: sygo1 ***@***.***>; Author ***@***.***>
Betreff: Re: [GrandOrgue/grandorgue] WS.Reputation.1 Bedrohung (Issue #1438)

 

eher ein Fehler bei NORTON als bei grandorgue - sofern exe oder zip von hier verwendet wird
